packages/amplify-e2e-core/src/utils/hooks.ts

```diff
@@ -0,0 +1,5 @@
+import * as path from 'path';
+
+export const getHooksDirPath = (projRoot: string): string => {
```

Comment:
isn't this already in pathManager?

Reply:
its in the e2e-core for not using path manager and making sure the path in path manager is not changed.